backend/core/types/src/backend-swagger.ts

```diff
@@ -89,40 +89,45 @@ export class PagedResult<T> implements IPagedResult<T> {
 // customer definition
 // empty
 
-export class AuthService {
```

Comment:
see PR description -- running `yarn  generate:client` resulted in lotso diff spam